scripts/mockapi.js

```diff
@@ -103,30 +105,17 @@ async function globalSearchAPI() {
   return fetchDataFromAPI(`${getHostUrl()}/scripts/mock-global-search.json`);
 }
 
-const fetchDynamicStockIndexData = () => [
-  {
-    id: 'spnNifty_n',
-    indexName: 'NIFTY',
-    stockValue: 22415.15,
-    change: 104.13,
-    changePercentage: 0.35,
-  },
-  {
-    id: 'spnSensex_s',
-    indexName: 'SENSEX',
-    stockValue: 73038.14,
-    change: -145.78,
-    changePercentage: -0.45,
-  },
-];
+function getGAToken() {
+  return fetchPlaceholders.GA_Token;
```

Comment:
Are we okay with keeping the GA_Token in the placeholder Excel? I see this excel as something where authors would change the displayed text also is it okay to expose the GA token since placeholder.json is published.

Reply:
yes, this is for now till they provide any way to get the token from the API